### PR TITLE
SoFiA-2

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
@conda-forge/help-c-cpp please review

**Comment by sebastian-luna-valero** [comment]
@conda-forge-admin, please ping team

File: recipes/SoFiA-2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: d6c43cc6fd2004bfc4ef410aecb34bd334b2ea753f7a237055ec68339e5ab212
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
```
Comment (chrisburr) [inline recipes/SoFiA-2/meta.yaml:19]:
```suggestion
    - make
    - sed
```

File: recipes/SoFiA-2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set name = "SoFiA-2" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SoFiA-Admin/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d6c43cc6fd2004bfc4ef410aecb34bd334b2ea753f7a237055ec68339e5ab212
```
Comment (chrisburr) [inline recipes/SoFiA-2/meta.yaml:10]:
```suggestion
  - url: https://github.com/SoFiA-Admin/{{ name }}/archive/v{{ version }}.tar.gz
    sha256: d6c43cc6fd2004bfc4ef410aecb34bd334b2ea753f7a237055ec68339e5ab212
    folder: sources
  - url: https://github.com/SoFiA-Admin/SoFiA-2/wiki/documents/sofia_test_datacube.tar.gz
    sha256: c4af151d8d0ef01daf9be06655a625ce6d821bec0361f7541b732edce932c1a7
    folder: test-data
```

File: recipes/SoFiA-2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - wcslib
+    - openmp
+  run:
+    - wcslib
+    - _openmp_mutex
+
+test:
+  commands:
+    - curl -LO https://github.com/SoFiA-Admin/SoFiA-2/wiki/documents/sofia_test_datacube.tar.gz
+    - tar xzf sofia_test_datacube.tar.gz
```
Comment (chrisburr) [inline recipes/SoFiA-2/meta.yaml:30]:
```suggestion
```

File: recipes/SoFiA-2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    - {{ compiler('c') }}
+    - make
+  host:
+    - wcslib
+    - openmp
+  run:
+    - wcslib
+    - _openmp_mutex
+
+test:
```
Comment (chrisburr) [inline recipes/SoFiA-2/meta.yaml:27]:
```suggestion
test:
  source_files:
    - test-data/
```

File: recipes/SoFiA-2/build.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
```
Comment (chrisburr) [inline recipes/SoFiA-2/build.sh:2]:
```suggestion

cd sources
```

**Comment by sebastian-luna-valero** [comment]
Thanks for your review @chrisburr 

The test fails because the command cannot find the input data as defined in the `meta.yaml`:
```
test:
  source_files:
    - test-data/
  commands:
    - sofia sofia.par
```
Do I need to change it to:
```
test:
  source_files:
    - test-data/
  commands:
    - cd test-data
    - sofia sofia.par
```
Best regards,
Sebastian

**Comment by chrisburr** [comment]
Yup sorry about that! 😅 LGTM

**Pull request merged**